frontend/src/components/AdminTable/index.jsx

```diff
@@ -9,6 +9,7 @@ import {
 } from "react-bootstrap";
 import Select from "react-dropdown-select";
 import CodebookLabelMenuContainer from "../../containers/codebookLabelMenu_container";
+import AnnotateCard, { buildCard } from "../AnnotateCard";
```

Comment:
Looks great! My only issue is I noticed we are getting some linting errors from the AdminTable and Skew modules. I'm not sure when these were introduced, but we should probably clear them out. 

To reproduce the errors, run `docker-compose -f envs/dev/docker-compose.yml run --rm smart_frontend yarn lint`

```
/code/src/components/AdminTable/index.jsx
   5:5  warning  'Button' is defined but never used          no-unused-vars
   6:5  warning  'ButtonToolbar' is defined but never used   no-unused-vars
   7:5  warning  'Tooltip' is defined but never used         no-unused-vars
   8:5  warning  'OverlayTrigger' is defined but never used  no-unused-vars
  10:8  warning  'Select' is defined but never used          no-unused-vars

/code/src/components/Skew/index.jsx
  63:13  warning  'labelsOptions' is assigned a value but never used  no-unused-vars

✖ 6 problems (0 errors, 6 warnings)
```

Reply:
fixed

Reply:
Great. Thanks!